crates/forge/bin/cmd/debug.rs

```diff
@@ -33,6 +33,15 @@ pub struct DebugArgs {
     #[arg(long)]
     pub debug: bool,
 
+    /// File path to dump execution details as JSON.
+    #[arg(
+        long,
+        requires = "debug",
```

Comment:
This syntax is a bit strange when using `forge debug` as it forces the user to add a `--debug` flag when dumping. When accessed from the `forge debug` command this should be implicit.

```
$ forge debug script/Counter.s.sol --dump dump.json --debug
```

Reply:
cc @piohei